src/components/Popover/__tests__/Popover.test.tsx

```diff
@@ -0,0 +1,299 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import '../../../tests/utils/fakeTimers';
+import {delayByBehavior, PopoverBehavior} from '../config';
+import {Popover} from '../Popover';
+import type {PopoverProps} from '../types';
+
+const defaultTooltipContent = 'Tooltip';
+const defaultTriggerText = 'Trigger';
+
+const renderPopover = (props?: Partial<PopoverProps>) => (
+    <Popover
+        content={defaultTooltipContent}
+        behavior={PopoverBehavior.Delayed}
+        qa="popover"
+        openOnHover={false}
+        {...props}
+    >
+        {defaultTriggerText}
+    </Popover>
+);
+
+const waitForTooltipOpenedStateChange = (shouldOpen?: boolean) =>
+    jest.advanceTimersByTime(delayByBehavior[PopoverBehavior.Delayed][shouldOpen ? 0 : 1]);
+
+const checkIfPopoverOpened = () => {
+    const popover = screen.queryByTestId('popover-tooltip');
+
+    expect(popover).toHaveClass('yc-popup_open');
```

Comment:
I suggest to check that popover is visible instead

Reply:
I've tried that, but it's not working